### ISSUE TITLE
Resume SQL fails with NPE

### ISSUE THREAD

**Issue by ideoma** [questdb/questdb#3058]
### Describe the bug

```
2023-03-10T10:07:39.458015Z I i.q.c.h.p.QueryCache miss [thread=questdb-shared-3, sql=alter table table_name resume wal from txn 123]
2023-03-10T10:07:39.458091Z I i.q.c.p.WriterPool >> [table=`table_name~15`, thread=14]
2023-03-10T10:07:39.458136Z I i.q.c.p.WriterPool << [table=`table_name~15`, thread=14]
2023-03-10T10:07:39.458213Z C i.q.c.h.p.JsonQueryProcessorState [27] Uh-oh. Error!
java.lang.NullPointerException
at io.questdb.cutlass.http.processors.JsonQueryProcessor.compileQuery(JsonQueryProcessor.java:421)
at io.questdb.cutlass.http.processors.JsonQueryProcessor.execute0(JsonQueryProcessor.java:184)
at io.questdb.cutlass.http.processors.JsonQueryProcessor.onRequestComplete(JsonQueryProcessor.java:234)
at io.questdb.cutlass.http.HttpConnectionContext.handleClientRecv(HttpConnectionContext.java:600)
at io.questdb.cutlass.http.HttpConnectionContext.handleClientOperation(HttpConnectionContext.java:203)
at io.questdb.cutlass.http.HttpServer$1.lambda$$0(HttpServer.java:86)
at io.questdb.network.AbstractIODispatcher.processIOQueue(AbstractIODispatcher.java:178)
at io.questdb.cutlass.http.HttpServer$1.run(HttpServer.java:101)
at io.questdb.mp.Worker.run(Worker.java:118)
```

### Environment

```markdown
- **QuestDB version**:7.0.1
- **OS**:
- **Browser**:
```

**Comment by hasseily** [comment]
Environment is QuestDB version 7.0.0

**Comment by ideoma** [comment]
Fixed in 7.0.1
